less/inc/widgets.less

```diff
@@ -210,7 +210,7 @@
   }
 }
 .single.normal #sidebar .widget {
-  padding: 0 @baseline 0 0;
+  padding: 0 (@baseline / 2);
```

Comment:
I think `.normal` is the class for our one-column layout, with `.classic` the two-column class, but I may be wrong here.